src/login.rs

```diff
@@ -277,7 +277,7 @@ impl Login {
                         192,
                         gdk::MemoryFormat::R8g8b8,
                         &bytes,
-                        1,
+                        192 * 3,
```

Comment:
I'd suggest creating a size variable before this line and using that instead.

Reply:
Can you see if it's the way you meant it? Else, feel free to unresolve.